lib/ReactCrop.js

```diff
@@ -67,6 +67,20 @@ class ReactCrop extends Component {
 		document.addEventListener('mouseup', this.onDocMouseTouchEnd);
 		document.addEventListener('touchend', this.onDocMouseTouchEnd);
 		document.addEventListener('touchcancel', this.onDocMouseTouchEnd);
+		
+		if (this.refs.image.complete || this.refs.image.readyState) {
+		  if (this.refs.image.naturalWidth === 0) {
+		    // Broken load
+		    // https://css-tricks.com/snippets/jquery/fixing-load-in-ie-for-cached-images/
+		    // http://stackoverflow.com/questions/821516/browser-independent-way-to-detect-when-image-has-been-loaded 
+		
+		    const src = this.refs.image.src;
+		    this.refs.image.src = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///ywAAAAAAQABAAACAUwAOw==';
```

Comment:
This comes up in IE. I put this in specifically for IOS. On refreshing the same page multiple times, and then using nartualWidth to crop I got 0 sometimes as a value. Then that affected the crop. 

From my understanding, that is because the cached image load was broken.